includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -659,10 +659,16 @@ public function register_scripts() {
 			true
 		);
 
+		$script_dependencies = [ 'stripe', 'wc-checkout' ];
+
+		if ( $this->supports( 'tokenization' ) ) {
+			$script_dependencies = array_merge( $script_dependencies, [ 'woocommerce-tokenization-form' ] );
```

Comment:
Aha I see - the tokenization script is not always enqueued. 

https://github.com/Automattic/woocommerce-payments/search?q=tokenization_script

The logic in each payment method is fairly consistent. Note that SEPA doesn't include the `|| is_add_payment_method_page()`

`$this->supports( 'tokenization' ) && ( is_checkout() || is_add_payment_method_page() )`